src/java/picard/illumina/IlluminaBasecallsConverter.java

```diff
@@ -453,8 +453,8 @@ public synchronized void addRecord(final String barcode, final CLUSTER_OUTPUT_RE
 
         private synchronized SortingCollection<CLUSTER_OUTPUT_RECORD> newSortingCollection() {
             final int maxRecordsInRam =
-                    maxReadsInRamPerTile /
-                            barcodeRecordWriterMap.size();
+                    Math.max(1, maxReadsInRamPerTile /
```

Comment:
I want it to be small so that the tests spill to disk, it was previously unprotected and becomes 0 due to integer division...this bumps that 0 up to 1.